dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/api/OpenTelemetryAccessor.java

```diff
@@ -40,6 +40,8 @@ public class OpenTelemetryAccessor {
     /**
      * Gets or creates the instance of OpenTelemetry associated with this application and returns it wrapped inside an OpenTelemetryInfo.
      *
+     * This method requires application metadata to be on the thread.
+     *
      * @return An instance of OpenTelemetryInfo containing the instance of OpenTelemetry associated with this application. This instance will be a no-op OpenTelemetry if telemetry
      *         is disabled or the application has shut down.
      */
```

Comment:
All classes which are API/SPI will have their [javadoc published on openliberty.io][javadoc]. Therefore we need to make sure we do proper HTML formatting and that all the parameters are documented.

Eclipse will do a decent preview of the formatted text if you hover over the documented element so you can check you have paragraph breaks where you expect them.

Following [Oracle's style guide][style-guide] is not a requirement, but it's a helpful guide to writing javadoc that's intended to be published wider than our team.

[style-guide]: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
[javadoc]: https://openliberty.io/docs/latest/reference/spi/open-liberty-spis.html

Reply:
An API package should also have a `package-info.java` explaining what the package allows you to do and usually giving an example of how to start using it.